src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs

```diff
@@ -6,13 +6,13 @@ namespace Tizen.NUI
 {
     internal static partial class Interop
     {
-        internal static partial class FrameCallbackInterface
+        internal static partial class FrameUpdateCallbackInterface
         {
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_new_FrameCallbackInterface")]
-            public static extern global::System.IntPtr new_FrameCallbackInterface();
+            public static extern global::System.IntPtr new_FrameUpdateCallbackInterface();
```

Comment:
if possible, could you remove this "\_" in name

Reply:
Thank you I fixed it.